checks.d/postgres.py

```diff
@@ -354,6 +358,25 @@ def _get_replication_metrics(self, key, db):
             metrics = self.replication_metrics.get(key)
         return metrics
 
+    def _build_relations_config(self, yc):
+        """Builds a dictionary from relations configuration while maintaining compatibility
+        """
+        config = {}
+        for x in yc:
+            try:
+                if isinstance(x, str):
+                    config[x] = {'relation_name': x, 'schemas': []}
+                elif isinstance(x, dict):
+                    name = x['relation_name']
+                    config[name] = {}
+                    config[name]['schemas'] = x['schemas'].split()
+                    config[name]['relation_name'] = name
+                else:
+                    self.log.warn('Unhandled relations config type: %s' % str(x))
+            except:
```

Comment:
bare:
```python
except:
```
should almost always be avoided as they would catch  SystemExit, KeyboardInterrupt, and GeneratorExit as well which you don't want. See https://docs.python.org/2/howto/doanddont.html#except for more details.

You should catch the specific exceptions that can be expected, or if they are too many, you should do a:
```python
except Exception:
```